hooks/bundle_init.py

```diff
@@ -12,6 +12,7 @@
 Hook that gets executed every time a bundle is fully initialized.
 """
 
+from __future__ import absolute_import
```

Comment:
For the future repos, I think we should do an early pass where we run the modernize scripts that puts the extra imports, fixes prints and exception syntax and merge those to master so the actual review is for the real work.

Reply:
👍